qml/ConnectionMenu.qml

```diff
@@ -175,20 +175,25 @@ Item {
 
                 Text {
                     text: "UDP Host/Port:"
-                    color: Material.primary
+                    color: udpIp.isValid ? Material.primary : Material.color(Material.Red)
```

Comment:
```suggestion
					color: udpIp.isValid ? Material.primary : Material.color(Material.Error)
```

Reply:
Nice. I was actually looking for that. Where can I find these colors?

Reply:
https://material.io/design/color/#color-theme-creation